README.md

```diff
@@ -14,3 +15,28 @@ part of the array each recursive call considers.
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+This implementationg of merge sort has 3 compnents:
+Outer for loop, inner while loop, and merge.
+
+Since the merge function, in essence, contains a loop, we can analyze the worst case scenerio
+by using loop logic. Ex: a loop inside a loop is equivilant to n*n.
+
+The outer for loop runs while size < the array length and since size is doubled each iteration and this loop 
+is reminicent of an exponential, which results in log(n) run time.
+
+The inner while loop runs while j < the array length and will happen length/2*size times,
+which results in n runtime. Therefore these loops have a runtime of nlog(n).
+
+The final piece is merge which has 2 loops. The outer while loop runs while i is less then j or
+essentially n/size times as it runs once for each array needing to be "merged". As we are analyzing the worst case
+scenerio, the inner loop will run essentially every time as no elements are sorted in our array. This results
+in the inner for loop running as long as k <= size + j - 1 and since k is incremented by 1 each time, it will run
+size + j - 1 times. Size and j are both elements of O(n) so this loop will run O(n) times in the wost case. This
+means that in the worst case the merge function runs n^2 times, as we multiply the loops together.
+
+Now we multiply the merge function with the outer loops, since it is in essense a loop as well, resulting in a 
+total runtime of $\Theta (n^3log(n))$
```

Comment:
Still don't see how you get to that. Please be explicit -- don't say just "the merge function" but also what complexity it is and what you're multiplying.